source/concepts.tex

```diff
@@ -1138,8 +1138,8 @@
 \pnum
 \begin{note}
 The \libconcept{semiregular} concept is modeled by types that behave similarly
-to built-in types like \tcode{int}, except that they might not
-be comparable with \tcode{==}.
+to built-in types like \tcode{int}, except that they are not necessarily
```

Comment:
I think I'd prefer "they need not be comparable" to "they are not necessarily comparable" here, but it's not a huge issue.

Reply:
fixed + force-pushed